luixxiul/browser-laptop
# Polish braveryPanel with Aphrodite

*`braveryPanel-aphrodite-for-PR` is a preparatory branch for the final PR.*

Test Plan 1:
1. Run tests locally; npm run test -- --grep='Bravery Panel'
2. Open about:preferences#shields
3. Enable the compact panel option
4. Run manual tests described here: https://github.com/brave/browser-laptop/wiki/Manual-Tests#bravery-settings

Test Plan 2:
1. Open about:styles
2. Click "Dropdowns"
3. Make sure BraveryPanelDropdown is displayed
